File: LastSemesterInfo/AYDA Laboratorio/AYDA_Examen_1_student_distribution_(A2026)/windows/planning_a_company_party/python/template.py
import sys

sys.setrecursionlimit(2000)

class Employee:
    def __init__(self, rating, left_child, right_sibling):
        self.rating = rating
        self.left_child = left_child
        self.right_sibling = right_sibling

def planParty(tree):
    return 0.0

if __name__ == '__main__':
    lines = sys.stdin.read().split()
    if not lines:
        sys.exit()

    n = int(lines[0])
    tree = []
    idx = 1
    for _ in range(n):
        rating = float(lines[idx])
        lc = int(lines[idx+1])
        rs = int(lines[idx+2])
        tree.append(Employee(rating, lc, rs))
        idx += 3

    result = planParty(tree)

    with open('solution.txt', 'w') as f:
        f.write(f"{result:.2f}\n")

File: LastSemesterInfo/AYDA Laboratorio/AYDA_Examen_1_student_distribution_(A2026)/windows/planning_a_company_party/cpp/template.cpp
#include <iostream>
#include <vector>
#include <fstream>
#include <iomanip>

using namespace std;

struct Employee
{
    double rating;
    int left_child;
    int right_sibling;
};

double planParty(const vector<Employee>& tree)
{
    return 0.0;
}

int main()
{
    int n;
    if (!(cin >> n)) return 0;

    vector<Employee> tree(n);
    for (int i = 0; i < n; ++i)
    {
        cin >> tree[i].rating >> tree[i].left_child >> tree[i].right_sibling;
    }

    double result = planParty(tree);

    ofstream fout("solution.txt");
    fout << fixed << setprecision(2) << result << "\n";
    fout.close();
    return 0;
}